vcbuild.bat

```diff
@@ -337,7 +337,8 @@ if errorlevel 1 (
 if "%target%" == "Clean" goto exit
 
 :after-build
-rd %config%
+:: Suppress "The system cannot find the file specified." on every clean build.
+rd %config% > nul 2> nul
```

Comment:
Would something like this work instead to not suppress other errors?

```suggestion
if exists %config% rd %config%
```

Reply:
The correct syntax should be 
"IF EXIST file command". However, check whether the file exists doesn't help a lot here because it's safe to remove the link even the link doesn't exists. Both of them works, and RD directly is just more simpler IMO.

Reply:
On the other hand, if `%config%` doesn't exist, it's not necessary to invoke `rd`, and when it does exist, this approach doesn't silently ignore the output of `rd`.

Reply:
Yes, I updated this pr. Thanks! 😄